src/C/cpl_vsi.jl

```diff
@@ -1135,28 +1339,25 @@ function VSITime(arg1)
     ccall((:VSITime, libgdal), Culong, (Ptr{Culong},), arg1)
 end
 
-
 """
     VSICTime(unsigned long nTime) -> const char *
 """
-function VSICTime(arg1::Culong)
+function VSICTime(arg1)
     ccall((:VSICTime, libgdal), Cstring, (Culong,), arg1)
 end
 
-
 """
     VSIGMTime(const time_t * pnTime,
               struct tm * poBrokenTime) -> struct tm *
 """
 function VSIGMTime(pnTime, poBrokenTime)
-    ccall((:VSIGMTime, libgdal), Ptr{Cvoid}, (Ptr{time_t}, Ptr{Cvoid}), pnTime, poBrokenTime)
+    ccall((:VSIGMTime, libgdal), Ptr{Ctm}, (Ptr{Cint}, Ptr{Ctm}), pnTime, poBrokenTime)
```

Comment:
What does `Ctm` stand for?

Reply:
Don't know what it stands for, but this is the definition. It's one of the ones that Clang.jl always writes:
const Ctm = Base.Libc.TmStruct